src/loader/InternalLoader.js

```diff
@@ -355,7 +355,7 @@ export class InternalLoader {
   }
 
   getOrCreateCodeUnit_(name, referrerName, address, metadata) {
-    var normalizedName = ModuleStore.normalize(name, referrerName, address);
+    var normalizedName = System.normalize(name, referrerName, address);
```

Comment:
I added info to the commit description. The change above worked but it was wrong.  We should be calling `this.loader_.normalize()`.  `System` would give the same result but we should not be going through global.